expressEngine/test/user/post/user.spec.js

```diff
@@ -0,0 +1,361 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../../../server';
+import { error, success } from '../../data';
+
+const { expect } = chai;
+const should = chai.should();
+chai.use(chaiHttp);
+
+describe('UserPost', () => {
+  describe('Post function in api/user/', () => {
+    it('register a user using given email and password', async () => {
```

Comment:
I think `it('should register a user using/with the given email and password', async () => {` is better than `it('register a user using given email and password', async () => {`